README.md

```diff
@@ -135,7 +135,7 @@ I would like to thank to my coding partners.
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 
```

Comment:
![image](https://github.com/ibugithub/math-Magician/assets/93039359/ca0ead47-5102-4b2b-b4e9-544d7de09af2)

- [ ] It would be great if you fix the live demo link that is not showing us the live demo of the app 🤔 Kindly fix this link as well 👍 to make your readme descriptive enough 😊